v2.0/roles.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Manage Roles
+summary: Roles are SQL groups that contain any number of users and roles as members.
+toc: false
+---
+
+<span class="version-tag">New in v2.0:</span> Roles are SQL groups that contain any number of users and roles as members. To create and manage your cluster's roles, use the following statements:
+
+- [`CREATE ROLE` (Enterprise)](create-role.html)
+- [`DROP ROLE` (Enterprise)](drop-role.html)
+- [`SHOW ROLES`](show-roles.html)
+- [`GRANT`](grant.html)
```

Comment:
`GRANT <role>` and `REVOKE <role>` are also Enterprise only. The (annoyingly) similar `GRANT <privilege> TO <role>` and `REVOKE <privilege> FROM <role>` are not.

Reply:
I call out `GRANT role` and `REVOKE role` as enterprise only in the separate pages. Do you think I should note that here as well?